src/protocols/secure_channel/CASESession.cpp

```diff
@@ -359,47 +394,56 @@ CHIP_ERROR CASESession::SendSigmaR1()
     return CHIP_NO_ERROR;
 }
 
-CHIP_ERROR CASESession::HandleSigmaR1_and_SendSigmaR2(const System::PacketBufferHandle & msg)
+CHIP_ERROR CASESession::HandleSigmaR1_and_SendSigmaR2(System::PacketBufferHandle & msg)
```

Comment:
Wondering why the `const` was removed (here and at other message handlers)?

Reply:
This is because PacketBufferTLVReader Class' Init method requires a non-const PacketBufferHandle